configs/vue3.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: [
+    require.resolve('./javascript'),
+    'plugin:@typescript-eslint/recommended',
+    'plugin:vue/vue3-recommended',
+    'prettier',
```

Comment:
Isn't this a duplicate from the extends in `./javascript`?

Reply:
Yes and no. It's duplicate but it has to be in the end otherwise it'll get overridden by everything defined after it. And it needs to be in the end.